### PR TITLE
Edits to Imperva partner doc

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

**Comment by ebgitelman** [comment]
Please check capitalization of the word "agent." There is a product Imperva Agent, but there is not a product Imperva Data Security Fabric Agent. So I tried to capitalize and revise with this in mind, but please double check.

File: advocacy_docs/partner_docs/ImpervaDataSecurityFabric/05-UsingImpervaDataSecurityFabric.mdx [hunk 1]
```diff
@@ -3,61 +3,63 @@ title: 'Using'
...
+You can use Imperva agents within the Imperva Data Security Fabric solution to track and review the actions of employees and third parties to ensure they're complying with policies and procedures.
 
-## Set-up to Monitor Database Traffic with Imperva Data Security Fabric Agent
+### Setting up to monitor database traffic with Imperva Data Security Fabric agent
 
-The following steps need to be taken in order to set-up the Imperva Data Security Fabric Agent to monitor EDB Postgres Advanced Server traffic.
+Set up the Imperva Data Security Fabric agent to monitor EDB Postgres Advanced Server traffic.
 
-1. Install Imperva Data Security Fabric Agent.
+1. Install Imperva Data Security Fabric.
```
Comment (dwicinas) [inline advocacy_docs/partner_docs/ImpervaDataSecurityFabric/05-UsingImpervaDataSecurityFabric.mdx:28]: @ebgitelman Is it clear what you mean if you omit the word "agent"? To me it is not.

File: advocacy_docs/partner_docs/ImpervaDataSecurityFabric/07-Support.mdx [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-description: 'Details of the Support process and logging information'
+title: 'Support and logging details'
+description: 'Details of the support process and logging information'
 ---
 
 ## Support
 
-Technical support for the use of these products is provided by both EDB and Imperva.  A proper support contract is required to be in place at both EDB and Imperva.  A support ticket can be opened on either side to start the process.  If it is determined through the support ticket that resources from the other vendor is required, the customer should open a support ticket with that vendor through normal support channels.  This will allow both companies to work together to help the customer as needed.
+Technical support for the use of these products is provided by both EDB and Imperva. A support contract is must in place at both EDB and Imperva. You can open a support ticket with either company to start the process. If it's determined through the support ticket that resources from the other vendor are required, open a support ticket with that vendor through normal support channels. This approach allows both companies to work together to help you as needed.
```
Comment (dwicinas) [inline advocacy_docs/partner_docs/ImpervaDataSecurityFabric/07-Support.mdx:8]:
```suggestion
Technical support for the use of these products is provided by both EDB and Imperva. A support contract must be in place at both EDB and Imperva. You can open a support ticket with either company to start the process. If it's determined through the support ticket that resources from the other vendor are required, open a support ticket with that vendor through normal support channels. This approach allows both companies to work together to help you as needed.
```

File: advocacy_docs/partner_docs/ImpervaDataSecurityFabric/05-UsingImpervaDataSecurityFabric.mdx (hunk 1 above)
Comment (dwicinas) [inline advocacy_docs/partner_docs/ImpervaDataSecurityFabric/05-UsingImpervaDataSecurityFabric.mdx:28]:
```suggestion
1. Install Imperva Data Security Fabric agent.
```

**Review by dwicinas — DISMISSED** [review]

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**